doc/nrf/device_guides/working_with_nrf/nrf70/developing/powersave.rst

```diff
@@ -414,6 +417,10 @@ The following shell commands and network management APIs are provided for power
      - wifi twt teardown_all
      - Tear down all sessions
      - TWT operation TWT teardown all flows
+   * - net_mgmt(NET_EVENT_WIFI_TWT_SLEEP_STATE)
+     -
```

Comment:
Put a N/A here. Also maybe change the name of the column to `Wi-Fi shell command` ?

Reply:
If so, please fill in the **Command** column:
![image](https://github.com/nrfconnect/sdk-nrf/assets/51916561/f9c367d3-ab6c-4996-a29a-bb2e5d5b59ff)


with the text: wifi shell 
Unless, the word "command" is part of the command.